test/yb_test.c

```diff
@@ -0,0 +1,339 @@
+/**
+ * yb_test.c
+ * Copyright (c) YugaByte, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <math.h>
+#include <time.h>
+#include <errno.h>
+#include <string.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/resource.h>
+#include <unistd.h>
+
+#include <hdr/hdr_histogram.h>
+#include <hdr/hdr_histogram_log.h>
+
+#include "minunit.h"
+
+#if defined(_MSC_VER)
+#pragma warning(push)
+#pragma warning(disable: 4996)
+#endif
+
+int tests_run = 0;
+int yb_default_max = 16777215;
+int yb_default_bucket_factor = 16;
```

Comment:
Should be static vars, and in capital letters, like `TESTS_RUN`.

Reply:
I've checked and mu_run_test only works with non-static integer tests_run. I have made the other two static, however.